terraform/k8s-monitoring/env/dev/terraform.tfvars

```
@@ -0,0 +1,15 @@
+aws_region = "eu-south-1"
+env        = "dev"
+
+tags = {
+  CreatedBy   = "Terraform"
+  Environment = "dev"
+  Owner       = "PagoPA"
+  Source      = "https://github.com/pagopa/interop-analytics-deployment"
+}
+
+eks_cluster_name = "interop-eks-cluster-dev"
+
+sns_topic_name = "interop-platform-alarms-dev"
+
+cloudwatch_log_group_name = "/aws/eks/interop-eks-cluster-dev/application" #TOCHECK: Do deployments in dev-analytics namespace send logs to this log group? 
```

```suggestion
cloudwatch_log_group_name = "/aws/eks/interop-eks-cluster-dev/application"
```
Yes, the logging configuration and output log group is global across the namespaces.